Kernel/Arch/x86_64/Processor.cpp

```diff
@@ -1723,6 +1723,7 @@ void Processor::switch_context(Thread*& from_thread, Thread*& to_thread)
         "shrq $32, %%rbx \n"
         "movl %%ebx, %[tss_rsp0h] \n"
         "movq %[to_rsp], %%rsp \n"
+        "movq %%rbp, %[from_rbp] \n"
```

Comment:
The commit says `rsp` but `rbp` is updated here :)

Reply:
Fixed! :^)